### DISCUSSION TITLE
Why did the Pacman game appear?

### DISCUSSION THREAD

**Discussion by Mickael81** [fast4x/RiMusic#5076] [Q&A]
Hello,
Today, after opening the application, I discovered something strange by clicking several times on the RiMusic logo located at the top left.
Messages (not translated) appeared at the bottom of the screen and, by continuing to click on the logo, a page with the Pacman game appeared. Surprised, I wondered what it was doing there.
Is this normal?
![Screenshot_20241225_225901](https://github.com/user-attachments/assets/fa330817-5c4e-4ae5-9887-81d888cce1db)
![Screenshot_20241225_230355](https://github.com/user-attachments/assets/be15872f-19f9-46fa-9fc6-e017f6e1ede0)
![Screenshot_20241225_230018](https://github.com/user-attachments/assets/55a9ea03-b8a6-4507-a972-9266b886f5a0)

**Comment by Stzyxh** [comment]
It is a secret feature

**Comment by Chandramauli-Arm64** [comment]
**Yeah 😂 that's the funny and secret feature and _**
